src/App.js

```diff
@@ -2,21 +2,39 @@ import React from 'react'
 import { Route } from 'react-router-dom'
 import Home from './containers/home'
 import './App.css'
-import logo from './logo.svg'
+
+import AppBar from 'material-ui/AppBar'
+import Avatar from 'material-ui/Avatar'
+import {Card, CardActions, CardHeader, CardMedia, CardTitle, CardText} from 'material-ui/Card'
+import Checkbox from 'material-ui/Checkbox'
+import TextField from 'material-ui/TextField'
+
+import SearchIcon from 'material-ui/svg-icons/action/search'
+
+// Image importing would only work via require
+const Logo = require('./film-indy-logo.jpg')
 
 const App = () => (
   <div className="App">
-    <div className="App-header">
-      <img src={logo} className="App-logo" alt="logo" />
-      <h2>Film Indy Project</h2>
-    </div>
-    <div className="App-subheader">
-      <h3>Built with <a href="https://github.com/facebookincubator/create-react-app" className="App-docs">React,</a>
-        <a href="http://redux.js.org/" className="App-docs">Redux,</a>
-        <a href="https://firebase.google.com/docs/" className="App-docs">& Firebase</a>
-      </h3>
-    </div>
-    <Route exact path="/" component={Home} />
+      <AppBar
```

Comment:
Indentation here seems a little off.  

```
    <AppBar
      iconElementLeft={
        <div>
          <img src={Logo} className="logo" />
          <Card className="searchCard" style={{ width: 400 }}>
            <SearchIcon className="searchIcon" />
            <TextField
              className="searchField"
              hintText="Search FilmIndy"
              underlineFocusStyle={{ borderColor: '#38b5e6' }}
              floatingLabelFocusStyle={{ color: '#38b5e6' }}
            />
          </Card>
        </div>
      }
      iconElementRight={<Avatar className="accountIcon" src="https://goo.gl/ybdoo6" size={60} />}
      zDepth={2}
    />
```

Also I think were missing the image here in this PR. 

I don't think this branch has the changes in master with the linting code in it. So that makes sense.